client/packages/system/src/Stock/Components/InventoryAdjustment/InventoryAdjustmentModal.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC } from 'react';
```

Comment:
This file is just renamed from "InventoryAdjustmentForm", only minimal code changes in here.